AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec

```diff
@@ -14,5 +14,5 @@ Pod::Spec.new do |s|
 
   s.vendored_frameworks = 'AppCenterReactNativeShared/AppCenterReactNativeShared.framework'
 
-  s.dependency 'AppCenter/Core', '1.9.0'
+  s.dependency 'AppCenter/Core', '1.9.1-3'
```

Comment:
Temporary version until Native SDK released, revert that after native SDK released.